sources/packages/api/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt-request.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from "@nestjs/common";
+import { InjectLogger } from "../../common";
+import { LoggerService } from "../../logger/logger.service";
+import {
+  ConfigService,
+  DisbursementReceiptService,
+  ReportService,
+} from "../../services";
+import { ESDCFileHandler } from "../esdc-file-handler";
+import { DailyDisbursementUploadResult } from "./models/disbursement-receipt-integration.model";
+import { ReportsFilterModel } from "src/services/report/report.models";
+import { DisbursementReceiptIntegrationService } from "./disbursement-receipt-integration.service";
+
+@Injectable()
+export class DisbursementReceiptRequestService extends ESDCFileHandler {
```

Comment:
Is there a reason to inherit from `ESDCFileHandler`?

Reply:
yes inherited as part of previous implementation, removed now.